static/css/style.css

```diff
@@ -183,6 +183,29 @@ figure{
     display: inherit;
 }
 
+/* Accessibility Page */
+.card-title {
+    font-size: xx-large;
+    text-align: center;
```

Comment:
Consider removing this and replacing it with the utitlity class from Bulma docs to center the text?